<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box_initvals">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="l_initvals">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lorentzian fit values</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_check_initvals">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Check initial values</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_int_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit3_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dBu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_init_off">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_int">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>54</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Intensity</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>54</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz*dBu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_init_int">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>54</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>107</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>kHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>82</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_lw">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>107</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Linewidth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_init_lw">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>107</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_pos">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>82</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_init_pos">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>82</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_fitrange_l">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>132</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>132</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MHz</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_fitrange">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>132</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fit range</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_fitrange_u">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>132</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit2_3">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>132</y>
       <width>20</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_scaninfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="l_measinfo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Measurement information</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_dt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Time between waveforms</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_dt">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_unit4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_start_fitting">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start fitting</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pB_fitting">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="PlotWidget" name="frame_spectrum" native="true">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="pB_import">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plot_results">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load and plot results</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="frame_frequency" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_choose_directory">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose data directory</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="frame_linewidth" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="frame_powerspectrum" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="frame_allandev" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>621</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
